docs-source/content/userguide/managing-domains/accessing-the-domain/admin-console.md

```diff
@@ -1,6 +1,101 @@
+
+1. Install, configure, and start the Remote Console according to these [instructions](https://github.com/oracle/weblogic-remote-console/blob/master/site/install_config.md).
+
+1. To give the Remote Console access to an Administration Server running in Kubernetes, you can:
+
+   * Use an [Administration Server `NodePort`](#use-an-administration-server-nodeport).
+
+   * Deploy a load balancer with [ingress path routing rules](#configure-ingress-path-routing-rules).
+
+
+#### Use an Administration Server `NodePort`
+
+For the Remote Console to connect to the Kubernetes WebLogic Server Administration Server’s `NodePort`, use the URL:
+
+```
+http://hostname:adminserver-NodePort/
+```
+
+The `adminserver-NodePort` is the port number of the Administration Server outside the Kubernetes cluster.
+For information about the `NodePort` Service on an Administration Server, see the [Domain resource](https://github.com/oracle/weblogic-kubernetes-operator/blob/master/docs/domains/Domain.md) document.
+
+{{% notice warning %}}
+Exposing administrative, RMI, or T3 capable channels using a Kubernetes `NodePort`
+can create an insecure configuration. In general, only HTTP protocols should be made available externally and this exposure
+is usually accomplished by setting up an external load balancer that can access internal (non-`NodePort`) services.
+For more information, see [T3 channels]({{<relref "/security/domain-security/weblogic-channels#weblogic-t3-channels">}}).
+{{% /notice %}}
+
+#### Configure ingress path routing rules
+
+1. Configure an ingress path routing rule. For more information about ingresses, see the [Ingress]({{< relref "/userguide/managing-domains/ingress/_index.md" >}}) documentation.
+
+   For an example, see the following `path-routing` YAML file for a Traefik load balancer:
+
+   ```yaml
+   apiVersion: traefik.containo.us/v1alpha1
+   kind: IngressRoute
+   metadata:
+     annotations:
+       kubernetes.io/ingress.class: traefik
+     name: traefik-pathrouting-1
+     namespace: weblogic-domain
+   spec:
+     routes:
+     - kind: Rule
+       match: PathPrefix(`/`)
+       services:
+       - kind: Service
+         name: domain1-adminserver
+         namespace: weblogic-domain
+         port: 7001
+   ```
+
+1. For the Remote Console to connect to the Kubernetes WebLogic Server Administration Server, use the URL:
+
+   ```
+   http://${HOSTNAME}:${LB_PORT}/
+   ```
+   Where:
+
+     * `${HOSTNAME}` is where you start up the WebLogic domain.
+
+     * To determine the `${LB_PORT}` when using a Traefik load balancer:
+
+        `$ export LB_PORT=$(kubectl -n traefik get service traefik-operator -o jsonpath='{.spec.ports[?(@.name=="web")].nodePort}')`
+
+### Test
+
+To verify the connection and that the correct `hostname:port` is being used, use the following curl command to access the REST interface of WebLogic Server Administration Server:
```

Comment:
_The raw output from '/management/weblogic/latest/' is a little overwhelming in my opinion -- it consequently almost looks like an error message in of itself. It'd be nice to use a REST command that yields a more succinct result.  Plus I personally favor having a little more detail in the explanatory text.  

For example, this section could be rewritten as:_

To verify that your WebLogic Administration Server URL is correct, and to verify that that your Load Balancer or NodePort are working as expected, run the following curl commands at the same location as your browser. 

```
$ curl --user username:password \
     http://${HOSTNAME}:${LB_PORT}:7001/management/weblogic/latest/domainRuntime?fields=name\&links=none ; echo
$ curl --user username:password \
     http://${HOSTNAME}:${LB_PORT}:7001/management/weblogic/latest/serverRuntime?fields=name\&links=none ; echo
```

These commands access the REST interface of WebLogic Server Administration Server, which is similar to the way the  Remote Console accesses the same server.

If successful, the output from the two commands should be `{"name": "your-weblogic-domain-name"}` and `{"name": "your-weblogic-admin-server-name"}` respectively.  

> If you want to see the full content `domainRuntime` and `serverRuntime` beans, then rerun the commands but remove the `?fields=name\&links=none` that's appended at the end of each URL.